keps/sig-architecture/20191001-prevent-permabeta.md

```diff
@@ -0,0 +1,140 @@
+---
+title: Require Transition from Beta
+authors:
+  - "@deads2k"
+owning-sig: sig-architecture
+participating-sigs:
+reviewers:
+  - "@bgrant0607"
+  - "@liggitt"
+  - "@smarterclayton"
+approvers:
+  - TBD
+editor: TBD
+creation-date: 2019-10-01
+last-updated: 2019-10-01
+status: implementable
+see-also:
+replaces:
+superseded-by:
+---
+
+# Require Transition from Beta
+
+## Table of Contents
+
+A table of contents is helpful for quickly jumping to sections of a KEP and for highlighting any additional information provided beyond the standard KEP template.
+
+Ensure the TOC is wrapped with <code>&lt;!-- toc --&rt;&lt;!-- /toc --&rt;</code> tags, and then generate with `hack/update-toc.sh`.
```

Comment:
nit: delete these instructions

_aside: should we have better designations for template content?_